### ISSUE TITLE
Can any was successful till now in booking a slot for 18 +

### ISSUE THREAD

**Issue by jishnu-gv** [bombardier-gif/covid-vaccine-booking#31]

**Comment by xxxAng** [comment]
Am not able to running since morning for Kolkata region, but it seems this region is so busy.
The availability api has cached data upto 30 minutes and thus I am not getting any open slot and its getting booked even before this script can get the availabiliy.
:( :( :(

**Comment by jishnu-gv** [comment]
Hi,

I am running from morning , but never got a hit till now :( :( :(

**Comment by BumpyKnuckls** [comment]
@xxxAng @jishnu-gv Its possible that no new slots have been added for your locations. I would suggest to confirm the same from the Cowin website.

**Comment by xxxAng** [comment]
> @xxxAng @jishnu-gv Its possible that no new slots have been added for your locations. I would suggest to confirm the same from the Cowin website.

Nope, have checked manually there were small amounts of slots opened at least 5 to 10 times for Kolkata.
But i think the issue is cached data, this script first call the API to check for available slots, if it gets then only try to book.
For our case, the demand is so high thats its probably lasting for few seconds only.

If API would have given us real data probably could book. :(

I think its gonna work for less busy regions only to be honest. :)

**Comment by xxxAng** [comment]
Yup 100 slots probably within 10-15 seconds and by the time availability api is probably still showing 10 minutes old data.
